app/_kong_plugins/datakit/examples/conditionally-store-cached-items.yaml

```
@@ -0,0 +1,61 @@
+description: |
+  Conditionally fetch or store cache data.
+
+extended_description: |
+  This example leverages the `cache` and `branch` nodes to conditionally store or retrieve
+  cache items.
+
+title: Conditionally fetch or store cache data.
+
+config:
+  nodes:
+    - type: static
+      values:
+        key: cache key
+      name: CACHE_KEY_GET
+    - type: cache
+      input: CACHE_KEY_GET
+      ttl: 200
+      name: GET
+    - type: branch
+      then:
+        - SET
+    - EXIT_MISS
+  else:
+    - EXIT_HIT
+  input: GET.miss
+  name: branch
+    - type: call
+      method: GET
+      url: http://localhost:8080/anything
```

```suggestion
      url: "http://localhost:8080/anything"
```
Build error: `mapping values are not allowed in this context at line 28 column 11` - clearly it gets the line wrong, but i think it doesn't like the `:`